sklearn/linear_model/tests/test_coordinate_descent.py

```diff
@@ -366,6 +368,56 @@ def test_model_pipeline_same_as_normalize_true(LinearModel, params):
     assert_allclose(y_pred_normalize, y_pred_standardize)
 
 
+# FIXME: 'normalize' to be removed in 1.2
+@pytest.mark.filterwarnings("ignore:'normalize' was deprecated")
+@pytest.mark.parametrize(
+    "estimator, estimator_params, is_sparse",
+    [(LinearRegression, {}, True),
+     (LinearRegression, {}, False)
+     ]
+)
+def test_linear_model_sample_weights_normalize_in_pipeline(estimator,
+                                                           estimator_params,
```

Comment:
If `estimator_params` is always `{}` in this parametrized test, let's just remove it for the sake of simplicity.